CharlieBackend.AdminPanel/Controllers/CoursesController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using CharlieBackend.AdminPanel.Services.Interfaces;
+using CharlieBackend.Core.DTO.Course;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.AdminPanel.Controllers
+{
+    [Route("[controller]/[action]")]
+    public class CoursesController : Controller
+    {
+        private readonly ICourseService _courseService;
+
+        public CoursesController(ICourseService courseService)
+        {
+            _courseService = courseService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> AllCourses()
+        {
+            var courses = await _courseService.GetAllCoursesAsync();
+
+            return View(courses);
+        }
+
+        [HttpGet("{id}")]
+        public async Task<IActionResult> UpdateCourse(long id, UpdateCourseDto updateCourseDto)
+        {
+            await _courseService.UpdateCourse(id, updateCourseDto);
+
+            return RedirectToAction("AllCourses", "Courses");
+        }
+
+        [HttpGet("{id}")]
+        public async Task<IActionResult> DisableCourse(long id)
+        {
+            await _courseService.DisableCourseAsync(id);
+
+            return RedirectToAction("AllCourses", "Courses");
+        }
+    }
+}
```

Comment:
Add new line at the end of the file

Reply:
Solved https://github.com/ita-social-projects/WhatBackend/pull/415/commits/924fd1ff33cdddd18a82276c59a0ab9f6980e2c7